pkgs/by-name/gd/gdlauncher-carbon/package.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, appimageTools
+, fetchurl
+, graphicsmagick
+, makeWrapper
+, copyDesktopItems
+, libxcb
+, xdg-utils
+, electron
+, makeDesktopItem
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gdlauncher-carbon";
+  version = "2.0.1";
+
+  src = appimageTools.extract {
+    inherit (finalAttrs) pname version;
+    src = fetchurl {
+      url = "https://cdn-raw.gdl.gg/launcher/GDLauncher__${finalAttrs.version}__linux__x64.AppImage";
+      hash = "sha256-pQnLjJx+Gy++YYWXWfV3qAIiMKxbniWCAhKEgKg/vYA=";
+    };
+  };
+
+  nativeBuildInputs = [
+    graphicsmagick
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/gdlauncher-carbon/resources
+    cp -r $src/resources/{binaries,app.asar} $out/share/gdlauncher-carbon/resources/
+
+    install -D $src/@gddesktop.png $out/share/icons/hicolor/2048x2048/apps/gdlauncher-carbon.png
+
+    # Most desktops won't check the 2048x2048 folder, so make a smaller one as well
+    gm convert $src/@gddesktop.png -resize 512x512 icon_512.png
+    install -D icon_512.png $out/share/icons/hicolor/512x512/apps/gdlauncher-carbon.png
+
+    runHook postInstall
+  '';
+
+  postConfigure =
+    let
+      libPath = lib.makeLibraryPath [
+        libxcb
+      ];
+      binPath = lib.makeBinPath [
+        xdg-utils
+      ];
```

Comment:
I'm not 100% sure this is needed, but I'm assuming that app does depend on xdg-utils to open up browser pages, including for signin. Would have to test on a system without it though.